tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -20,6 +20,40 @@
     ("long_term_test_false", 15, long_term_list_reco_json_schema, LONG_TERM_DURATION_IN_HRS_MAX, False),
 ]
 
+term_input = [
```

Comment:
Include tests to validate short term recommendations not present and medium term recommendations are available and short term, medium term not present and long term is present.

Reply:
Done

Reply:
@khansaad - As discussed, please ensure all these combinations are covered, I will take a look too.

('L', 'M', 'S')
('L', 'M', 'NS')
('L', 'NM', 'S')
('L', 'NM', 'NS')
('NL', 'M', 'S')
('NL', 'M', 'NS')
('NL', 'NM', 'S')
('NL', 'NM', 'NS')

NL - No long term, NM - No medium term, NS - No short term